react-effects/my-app/src/List.tsx

```diff
@@ -0,0 +1,58 @@
+/* eslint-disable @typescript-eslint/no-unused-vars -- Remove me */
+import { useEffect, useState } from 'react';
+import { readItems, type Item } from './read';
+
+export function List() {
+  const [isLoading, setIsLoading] = useState(true);
+  const [items, setItems] = useState<Item[]>([]);
+  const [error, setError] = useState<unknown>();
+
+  // Your code here:
+  //  - When the component mounts:
+  //    - Read the items using `readItems` and update state so the list displays
+  //    - Handle errors from `readItems`
+
+  useEffect(() => {
+    let ignore = false;
```

Comment:
Looks like you got this from the React reading. They were trying to solve a problem with this that you don't need to worry about. You can remove `ignore` and all your uses of it.